test/distributed/fsdp/test_wrap.py

```diff
@@ -462,6 +462,17 @@ def test_module_wrap_policy(self):
         )
         self._test_transformer_wrapping(auto_wrap_policy)
 
+    @unittest.skipIf(not TEST_MULTIGPU, "Requires at least 2 GPUs")
+    def test_module_wrap_policy_callable(self):
+        """Tests the ``ModuleWrapPolicy`` as a ``Callable``."""
+        auto_wrap_policy = ModuleWrapPolicy(
+            {TransformerEncoderLayer, TransformerDecoderLayer}
+        )
+        callable_policy = functools.partial(
+            _or_policy, policies=[auto_wrap_policy]
```

Comment:
nit: iiuc _or_policy lets you use more than one policy and or's them? if so might be nice to test that by doing 

awp1 = ModuleWrapPolicy(encoder..)
awp2 = ... (decoder)

functools.partial(or, [policies=[awp1, awp2])

Reply:
I was not really trying to test _or_policy per se but am rather using it to "hide" the _Policy super type of ModuleWrapPolicy thus testing ModuleWrapPolicy as a Callable rather than as a _Policy.

There are already other unit tests which cover _or_policy. But if you prefer, I can change this unit test as you suggest to further test _or_policy.